components/buttons/button.css

```diff
@@ -41,29 +41,35 @@ color:var(--complement-btn-text-color);
 .btn-primary-outline{
     background-color:transparent;
     outline:1px solid var(--primary-color);
-    color:var(--complement-btn-text-color);
+    color:var(--text-color);
```

Comment:
you can select all the classes at once and then give them the desired color at once, Makes it easier to change the color of all the buttons in one go

Reply:
Actually, I have different colors for different classes. 
primary for primary-btn class, accent for accent-btn class that is why cannot combine them all together!

Reply:
Okay